relayer/src/chain/cosmos/compatibility.rs

```diff
@@ -23,7 +23,7 @@ const SDK_MODULE_NAME: &str = "cosmos/cosmos-sdk";
 /// # Note: Should be consistent with [features] guide page.
 ///
 /// [features]: https://hermes.informal.systems/features.html
-const SDK_MODULE_VERSION_REQ: &str = ">=0.41.3, <=0.42.6";
+const SDK_MODULE_VERSION_REQ: &str = ">=0.41.3, <=0.42.8";
```

Comment:
FYI a parallel PR updates this to 0.42.9 #1266.

Reply:
I've seen that one. I believe if that PR gets merged first then mine should be OK. I had to change it to ensure the testing works.

Reply:
Perfect, thanks for the feedback Andy! We'll try to merge this today.